Feature/doc improvements (#135)
w3c-ccg/traceability-vocab: README.md

# Traceability Vocabulary Specification

![CI](https://github.com/w3c-ccg/traceability-vocab/workflows/CI/badge.svg) [![CD](https://github.com/w3c-ccg/traceability-vocab/actions/workflows/cd.yml/badge.svg)](https://github.com/w3c-ccg/traceability-vocab/actions/workflows/cd.yml)

This specification describes a Linked Data vocabulary for asserting Verifiable Credentials related to traceability information, such as chemical properties, mechanical properties, country of origin, and other attributes used to determine the status of a products and materials in a supply chain.

We encourage contributions meeting the [Contribution Guidelines](CONTRIBUTING.md). While we prefer the creation of issues and Pull Requests in the GitHub repository, discussions often occur on the [public-credentials](http://lists.w3.org/Archives/Public/public-credentials/) mailing list as well.

## Getting Started

> If youare just interested in the vocabulary itself, the items contained within, and the example Verifiable Credentials, plase see the [spec itself](https://w3c-ccg.github.io/traceability-vocab/).

Reviewer: ```suggestion
If you are only interested in the vocabulary itself, the items contained therein, and/or the example Verifiable Credentials, please see the [spec itself](https://w3c-ccg.github.io/traceability-vocab/).
```

> If you are a developer who is interested in working directly with the vocabulary, possibly add schemas, as well as to run tests and view the details of how the JSON-JD and resultant Verifiable Credentials are build, then read on.

Reviewer: ```suggestion
If you are a developer who is interested in working directly with the vocabulary, possibly adding schemas, and/or running tests and viewing details of how the JSON-JD and resultant Verifiable Credentials are built, then read on.
```

### Developer Setup

> To get started with the repository you will need at a minimum [Node.js](https://nodejs.org/en/) and [git](https://git-scm.com/) in order to checkout the project, fetch other dependencies, and build and run the project.

Reviewer: ```suggestion
To get started with the repository -- to check-out the project, fetch other dependencies, and build and run the project -- at a minimum, you will need [`Node.js`](https://nodejs.org/en/) and [`git`](https://git-scm.com/).
```

Author: putting an alternate wording in

> To get started with the repository you will need at a minimum [Node.js](https://nodejs.org/en/) and [git](https://git-scm.com/) in order to checkout the project, fetch other dependencies, and build and run the project.

Author: ```suggestion
To get started building, testing, or contributing the repository you will need at a minimum [Node.js](https://nodejs.org/en/) and [git](https://git-scm.com/)
```

```
    "$comment": "{\"term\": \"Person\", \"@id\": \"https://schema.org/Person\"}",
```

Pull requests will not as a rule be merged if any conflicts exist, or if testing is incomplete.

Any changes that potentially introduce breaking or non-backwards compatible functionality MUST have a corresponing issue and discussion, and will require consensus from the editors in order to be introduced or to have any related Pull Requests accepted and merged.

A one week (7 day) period will be provided for review of pull requests related to data schemas or project functionality prior to merge to allow suffient review time. Execptions may be made for essential documentation, or to allow for immediate hotfix of security issues or functionality breaking items.

## Ontology Structure

> This repository hosts both [json-schema](https://json-schema.org/) which it uses to create [jsonld](https://json-ld.org/).

Author: ```suggestion
This repository hosts [json-schema](https://json-schema.org/) which it uses to create [jsonld](https://json-ld.org/).
```
